amazon-kinesis-client/src/main/java/software/amazon/kinesis/coordinator/Scheduler.java

```diff
@@ -99,6 +103,7 @@
 public class Scheduler implements Runnable {
 
     private static final long NEW_STREAM_CHECK_INTERVAL_MILLIS = 1 * 60 * 1000L;
+    private static final long OLD_STREAM_DEFERRED_DELETION_PERIOD_MILLIS = 1 * 60 * 60 * 1000L;
```

Comment:
```suggestion
    private static final long OLD_STREAM_DEFERRED_DELETION_PERIOD_MILLIS = Duration.ofHours(1).toMillis();;
```

Reply:
Ya more readable. Will update

Reply:
removed this constant in favor of dynamic configuration.